bootstrap.sh

```diff
@@ -6,37 +6,11 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 #
 # Use the --force option to force a re-build locales.
-# Use the --importLocales option to fetch and update locales only
```

Comment:
Just another comment @FilippoZazzeroni since you cleaned this bootstrap, is there anything in the wiki that needs to be updated? Just want to double check for instructions in readme, wiki etc.

Reply:
I didn't check it @nbhasin2, i'll be looking into it tomorrow morning first thing

Reply:
Thanks :)

Reply:
I didn't find anything related to bootstrap --importLocales on the wiki. i modified the link to `Strings.swift` that was outdated under *LocalizationTools* and *modify Strings*.